frontend/src/App.tsx

```diff
@@ -0,0 +1,78 @@
+import { useEffect, useState } from "react";
+
+import {
+  RenderEmbeddings,
+  Navbar,
+  EmbeddingDetails,
+  Footer,
+  Loader,
+} from "./components";
+import { getEmbeddingPoints, getEmbeddingPointDetails } from "./actions";
+import "./index.css";
+
+function App() {
```

Comment:
In a future diff I would suggest refactoring this into it's own component. So app can be used for higher level features such as router, header, footer etc.